packages/gradle/src/plugin/nodes.ts

```diff
@@ -127,309 +93,32 @@ export const makeCreateNodesForGradleConfigFile =
       options ?? {},
       context
     );
-    targetsCache[hash] ??= await createGradleProject(
-      gradleReport,
-      gradleFilePath,
-      options,
-      context,
-      gradleProjectRootToTestFilesMap[projectRoot]
-    );
+    targetsCache[hash] ??=
```

Comment:
super nit but why is this a targetsCache if it contains projects? Shouldn't it be a projectsCache?

Reply:
changed